.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,44 @@
-## Description
+**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be compeleted as written. Provide known gaps to testing that may raise the risk of merging to production.
 
 
-## Original issue(s)
-department-of-veterans-affairs/va.gov-team#0000
+## Summary
 
+- *(Summarize the changes that have been made to the platform)*
+- *(If bug, how to reproduce)*
+- *(What is the solution, why is this the solution)*
+- *(Which team do you work for, does your team own the maintainence of this component?)*
+- *(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)*
+
+## Related issue(s)
+
+- *Link to ticket created in va.gov-team repo*
+- *Link to previous change of the code/bug (if applicable)*
+- *Link to epic if not included in ticket*
 
 ## Testing done
 
+- *Describe what the old behavior was prior to the change*
+- *Describe the steps required to verify your changes are working as expected*
+- *Describe the tests completed and the results*
 
 ## Screenshots
 
+- [ ] *Add screenshot of current state prior to change*
+- [ ] *Add screenshot showing successful change and what should be expected of those verifying your changes work as intended*
```

Comment:
- Remove and replace with a table for before & after with mobile & desktop
- This field is mandatory for component work and UI changes (non-component work should NOT have screenshots).

| | Before | After |
| --- | --- | --- |
| Mobile | | |
| Desktop | | |